<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceLoginDialog</class>
 <widget class="QDialog" name="FaceLoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Authentification par Reconnaissance Faciale</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #f0f2f5;
}

QLabel {
    color: #1a1a1a;
    font-size: 14px;
}

QPushButton {
    background-color: #0078d4;
    color: white;
    border: none;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
    min-width: 100px;
}

QPushButton:hover {
    background-color: #005a9e;
}

QPushButton:pressed {
    background-color: #004578;
}

QPushButton#captureButton {
    background-color: #107c10;
    font-size: 16px;
    padding: 12px 24px;
}

QPushButton#captureButton:hover {
    background-color: #0b5c0b;
}

QLabel#statusLabel {
    color: #666666;
    font-size: 16px;
    margin: 10px;
}

QWidget#cameraFrame {
    border: 2px solid #cccccc;
    border-radius: 8px;
    background-color: #ffffff;
}

QFrame#mainFrame {
    background-color: #ffffff;
    border-radius: 10px;
    padding: 20px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 24px;
font-weight: bold;
color: #0078d4;
margin-bottom: 10px;</string>
     </property>
     <property name="text">
      <string>Reconnaissance Faciale</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="cameraFrame">
     <property name="minimumSize">
      <size>
       <width>640</width>
       <height>360</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>360</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid #cccccc;
border-radius: 8px;
background-color: #ffffff;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="styleSheet">
      <string notr="true">color: #666666;
font-size: 16px;
margin: 10px;</string>
     </property>
     <property name="text">
      <string>Positionnez votre visage dans le cadre</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="captureButton">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>40</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #107c10;
    color: white;
    border: none;
    border-radius: 4px;
    font-size: 16px;
    padding: 8px 16px;
}
QPushButton:hover {
    background-color: #0b5c0b;
}
QPushButton:pressed {
    background-color: #094509;
}</string>
       </property>
       <property name="text">
        <string>ðŸ“¸ Capturer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #d83b01;
    color: white;
    border: none;
    border-radius: 4px;
    font-size: 16px;
    padding: 8px 16px;
}
QPushButton:hover {
    background-color: #a32d01;
}
QPushButton:pressed {
    background-color: #7a2201;
}</string>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
